### ISSUE TITLE
Standards for DOM js bindings

### ISSUE THREAD

**Issue by iamjoshellis** [wearelighthouse/scss#6]
https://github.com/wearelighthouse/scss-lint/edit/feature-readme/README.md#L120

**Comment by RussellBishop** [comment]
Does this fit into the Linthouse? Or is this more for Filament...

**Comment by iamjoshellis** [comment]
I think this probably covers both this and js linting. i.e. if the standard is (what I'd suggest):

>  Use class names for styling only _not_ for js binding.

Then there probably should be a note in there in the 'selectors section'

**Comment by cjquinn** [comment]
Yeah I think this should just be mentioned where ever it is relevant which I imagine will be here and under the JavaScript style guide.

**Comment by cjquinn** [comment]
Can you link that article Josh?

**Comment by iamjoshellis** [comment]
https://toddmotto.com/data-js-selectors-enhancing-html5-development-by-separating-css-from-javascript/

TL;DR

```
  /**
   * Helper to get data-js attr with $
   */
  $.js = function(el) {
    return $('[data-js=' + el + ']');
  };
```

Usage:

```
<div data-js="gallery">
 ...
</div>
var $gallery = $.js('gallery');
```
